src/main/webapp/views/home.html

```diff
@@ -13,14 +14,43 @@
     <script src="../scripts/popper.min.js"></script>
     <script src="../scripts/bootstrap.bundle.min.js"></script>
 
+    <script src="../scripts/tutorial.js"></script>
     <script src="../scripts/auth.js"></script>
     <script src="../scripts/navbar.js"></script>
     <script src="../scripts/onload.js"></script>
   </head>
   <body>
-    <div id="home_container">
-      <div id="home_wrapper">
+    <div id="home_container" class="h-100">
+      <div id="home_wrapper" class="h-100">
         <div id="navbar_container"></div>
+        <br><br><br><br>
```

Comment:
Please use bootstrap for adding margins, you can see more about that [Here.](https://getbootstrap.com/docs/4.0/utilities/spacing/)
```suggestion
        
```

Reply:
Of course! I'll look more into it, thanks fro the recommendation.